randovania/games/prime1/exporter/patch_data_factory.py

```diff
@@ -802,7 +802,7 @@ def create_game_specific_data(self) -> dict:
         else:
             starting_memo = None
 
-        if self.cosmetic_patches.open_map and self.configuration.teleporters.is_vanilla:
+        if self.cosmetic_patches.open_map:
             map_default_state = "Always"
         else:
             map_default_state = "MapStationOrVisit"
```

Comment:
mentioned it in discord already, writing down here too: please add a test case that results in MapStationOrVisit being set.